test/gtest-extra.h

```diff
@@ -12,12 +12,7 @@
 
 #include <string>
 
-#ifdef FMT_MODULE_TEST
-import fmt;
-#else
-#  include "fmt/os.h"
-#endif  // FMG_MODULE_TEST
-
+#include "fmt/os.h"
```

Comment:
Why change this?

Reply:
With all compilers having converged to the 'strong module ownership' model we no longer need to fold module `fmt` into the test machinery for module tests. The module has *all* its symbols attached to the module (e.g. `void foo()` becomes `void foo@fmt()`), and the test machinery can have the same definitions attached to the global module **within the same program**.

Therefore, the test-main library can be reused as with all other tests. And all stuff conditionally compiled by `FMT_MODULE_TEST` can kicked out again. Both Clang and MSVC are fine with it. Strong ownership rulez!